.github/workflows/release-production.yml

```diff
@@ -4,6 +4,17 @@ on:
   push:
     tags: [ v* ]
 
+  workflow_dispatch:
+    inputs:
+      publish:
```

Comment:
Input unused.
You should make it so only the build process runs if `publish` is false

Reply:
Updated.